config/confd/templates/docker-registry.yml.tmpl

```diff
@@ -38,6 +41,3 @@ redis:
   addr: {{getv "/resin/registry2/cache-addr" "127.0.0.1:6379"}}
   db: {{getv "/resin/registry2/cache-db" "0"}}
 {{end}}
-
-redirect:
```

Comment:
Why is this removed?

Reply:
It never worked as intended — it’s specified in the wrong place and registry just ignores it.